transforms/language/html2parquet/dpk_html2parquet/transform.py

```diff
@@ -51,7 +50,7 @@ def __init__(self, config: dict[str, Any]):
         if not isinstance(self.favor_recall, html2parquet_favor_recall):
             self.favor_recall = html2parquet_favor_recall[self.favor_recall]
 
-    def _convert_html2parquet(self, member_filename: str, file_name: str, content_bytes: bytes) -> dict:
+    def convert_html2parquet(self, member_filename: str, file_name: str, content_bytes: bytes) -> dict:
```

Comment:
why the name change?  It is not used outside this file and so can be considered at least protected, if not private.  In general, methods should not be exposed if not necessary.  I know we're not consistent on this, but better to catch it when we can

Reply:
And sub-classes generally are considered to have access to their parent _ methods.

Reply:
I don't see the reason why this particular method was done as a private method in the first place. Once you create a valid object, it might be helpful to have access to this method.

Reply:
Always ere on the side of private/protected until someone asks for it.  Otherwise, you unnecessarily create additional support and backwards compatibility issues.  It unnecessarily ties our hands going forward.